study-builder/studies/testboston/i18n/en.conf

```diff
@@ -6,7 +6,7 @@
 
   "consent": {
     "name": "Consent Form",
-    "title": "Consent Process<a class="button button_download" href="consent.pdf" download><img src="assets/images/download.svg" alt="Download file" class="activity-header__icon">Download this document</a>",
+    "title": """Consent Process<a class="button button_download" href="consent.pdf" download><img src="assets/images/download.svg" alt="Download file" class="activity-header__icon">Download this document</a>""",
```

Comment:
It looks like we should put the title content in the additional quotes due to inner quotes.